recipe-client-addon/test/browser/browser_PreferenceExperiments.js

```diff
@@ -611,19 +647,36 @@ decorate_task(
     Assert.deepEqual(
       setActiveStub.getCall(0).args,
       ["test", "branch", {type: "normandy-exp"}],
-      "Experiment is registerd by start()",
+      "Experiment is registered by start()",
     );
-    await PreferenceExperiments.stop("test");
+    await PreferenceExperiments.stop("test", {reason: "test"});
     ok(setInactiveStub.calledWith("test", "branch"), "Experiment is unregistered by stop()");
+
+    Assert.deepEqual(
+      sendEventStub.getCall(0).args,
+      ["enroll", "preference-study", "test", {
+        experimentType: "exp",
+        branch: "branch",
+      }],
+      "start should send the correct telemetry event"
```

Comment:
I like to have unique failure messages so I can quickly find out where things failed, but it's up to you.